proto/cosmos/group/v1/query.proto

```diff
@@ -303,12 +306,14 @@ message QueryGroupsByMemberResponse {
 
 // QueryTallyResultRequest is the Query/TallyResult request type.
 message QueryTallyResultRequest {
+
```

Comment:
I think we should remove these new lines. As far as I can see message definitions in other files don't have this.

Reply:
Correct. Please remove new lines after type declaration.